lms/static/js/groups/views/cohorts.js

```diff
@@ -304,7 +304,7 @@
                                 });
                             }
                         }).render();
-                        this.$('#file-upload-form-file').focus();
+                        this.$('#file-upload-form-file');
```

Comment:
But we want the functionality of the item being focused (plus you left a nonsensical line of code in).